src/partials/head-info.hbs

```diff
@@ -1,5 +1,5 @@
     {{#with page.canonicalUrl}}
-    <link rel="canonical" href="{{{this}}}">
+    {{!-- <link rel="canonical" href="{{{this}}}"> --}}
```

Comment:
I think it's unnecessary, `page.canonicalUrl` will only be defined when the site URL is set to an absolute URL: https://docs.antora.org/antora/2.3/playbook/site-url/

We can either remove the block completely or leave it.
Please note that the following is a bit of a hack and ideally we should use the built-in `page.canonicalUrl` feature:


```hbs
<link rel="canonical" href="https://neo4j.com{{ page.attributes.canonical-root }}{{ page.url }}">
```

Reply:
Please advise on whether docs should be specifying an absolute URL in the playbook.